apps/src/sites/studio/pages/print_certificates/show.js

```diff
@@ -0,0 +1,6 @@
+import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
+import {EVENTS} from '@cdo/apps/lib/util/AnalyticsConstants';
+
+$(() => {
+  analyticsReporter.sendEvent(EVENTS.CERTIFICATE_PRINT_PAGE_VISITED);
+});
```

Comment:
Is this a good reference for how to add Amplitude events on studio.code.org pages?

Reply:
I would say so! Most dashboard pages have associated JS these days, so normally I wouldn't need to add a new file and would just add this event to the existing JS file. In this case, Turner and I debated a bit offline about whether to add a partial like we have for pegasus or just add the JS like this and settled on adding a JS file. Definitely open to push back here!

Reply:
Cool, ty! What was the reasoning for going with the JS file?

Reply:
We'd need to add a javascript entry point either way. The pegasus helper mostly [loads a JS file](https://github.com/code-dot-org/code-dot-org/blob/37147bc79aef7bf0c0fe83bc546f2de8b319da13/pegasus/sites.v3/code.org/views/analytics_event_log_helper.haml#L2). The thinking here was that we want dashboard pages to move away from haml in general, so most will already load some JS. There are obviously exceptions (older pages, pages like this one that's so simple it doesn't need much), but it didn't seem worth the effort to create a customizable view for those exceptions.